src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -426,7 +430,7 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRest
 
     [HttpPatch("enable-secrets-manager")]
     [HttpPut("enable-secrets-manager")]
-    public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkEnableSecretsManagerAsync(Guid orgId,
+    public async Task BulkEnableSecretsManagerAsync(Guid orgId,
```

Comment:
Are we going to have a client PR to clean up the `OrganizationUserServiceImplementation` expecting the list response?

Reply:
Great point! PR here: https://github.com/bitwarden/clients/pull/5924